arho_feature_template/core/geotiff_creator.py

```diff
@@ -125,7 +125,13 @@ def _create_geotiff_from_png(self, image_path, geotiff_path):
             ds,
             outputSRS="EPSG:3067",
             format="GTiff",
-            creationOptions=["COMPRESS=LZW", "TILED=YES"],
+            outputType=gdal.GDT_Byte,
+            bandList=[1, 2, 3],
+            creationOptions={
+                "COMPRESS=LZW",
+                "TILED=YES",
+                "BIGTIFF=IF_SAFER",
+            },
```

Comment:
creation options voisi dict, niin olisi pythonmaisempi. Voisi varmaankin olla listan ja dictin lisäksi set tai tuple, mutta ehkä dict selkein.
```suggestion
            creationOptions={
                "COMPRESS": "LZW",
                "TILED": "YES",
                "BIGTIFF": "IF_SAFER",
            },
```